mitmproxy/contentviews/img/metadata.py

```diff
@@ -0,0 +1,30 @@
+from . import png
```

Comment:
Yes, we'd need to move this into `mitmproxy.contentviews.image` as well, e.g. as `mitmproxy/contentviews/image/pillow.py`